neuro_san/test/assessor/assessor_assert_forwarder.py

```diff
@@ -42,199 +42,48 @@ def get_fail_dicts(self) -> List[Dict[str, Any]]:
         """
-        self.handle_assert(bool(expr), True, msg)
+        self.handle_assert(gist, acceptance_criteria, text_sample, True)
 
-    def assertFalse(self, expr: Any, msg: str = None):
+    # pylint: disable=invalid-name
+    def assertNotGist(self, gist: bool, acceptance_criteria: str, text_sample: str, msg: str = None):
         """
-        Assert that the expression is false
+        Assert that the gist is true
 
-        :param expr: Expression to test
+        :param gist: Pass/Fail value of the gist expected to be False
+        :param acceptance_criteria: The value to verify against
+        :param text_sample: The value appearing in the test sample
         :param msg: optional string message
         """
-        self.handle_assert(bool(expr), False, msg)
+        self.handle_assert(gist, acceptance_criteria, text_sample, False)
 
-    def handle_assert(self, is_passing: bool, sense: bool, msg: str):
+    def handle_assert(self, is_passing: bool,
+                      acceptance_criteria: str,
+                      text_sample: str,
+                      sense: bool):
         """
         Handle the assert.
         :param is_passing: Boolean as to whether or not the test is passing.
+        :param acceptance_criteria: The value to verify against
+        :param text_sample: The value appearing in the test sample
         :param sense: Whether the test was supposed to be true or false.
-        :param msg: The assert message to parse.
         """
         self.num_total += 1
         if is_passing == sense:
             return
 
-        components: Dict[str, Any] = self.parse_assert_message(msg)
-        components["sense"] = sense
-        self.fail.append(components)
-
-    def parse_assert_message(self, message: str) -> Dict[str, Any]:
-        """
-        Parse a single assert message into its parts.
-        :param message: The assert message from GistAgentEvaluator
-        :return: A dictionary with the relevant components of the message
-        """
-
-        # See GistAgentEvaluator for format.
-        acceptance_split: List[str] = message.split("acceptance_criteria:")
-        sample_split: List[str] = acceptance_split[0].split("text_sample:")
-
         components: Dict[str, Any] = {
-            # Get what comes after the "acceptance_criteria" delimiter
-            "acceptance_criteria": acceptance_split[-1],
-
-            # Get what comes after the "text_sample" delimiter
-            "text_sample": sample_split[-1]
+            "acceptance_criteria": acceptance_criteria,
+            "text_sample": text_sample,
+            "sense": sense
         }
-        return components
-
-    def assertEqual(self, first: Any, second: Any, msg: str = None):
-        """
-        Assert that the first is equal to the second
-
-        :param first: First comparison element
-        :param second: Second comparison element
-        :param msg: optional string message
-        """
```

Comment:
All these do-nothing assert handling is not in the realm of the NullAssertForwarder parent class.